recipe/meta.yaml

```diff
@@ -10,40 +10,49 @@ package:
 source:
   fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
-  sha256: 8f157e7397158812cabd340b68546f1baa55a486ed0aad8bc26877593dc2983d
+  sha256: 3296c9d49b7dcb894b20db5d7d1c1a443912b4d82358e03f836575e8398e0d60
 
 build:
-  skip: true  # [win]
-  number: 202
+  skip: true  # [win32 or (win and py<35)]
+  number: 200
   features:
     - blas_{{ variant }}
+    - vc14    # [win and py>=35]
 
 requirements:
   build:
     - python
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - gsl
-    - fftw
+    - gsl     # [not win]
+    - fftw    # [not win]
     - glpk
-    - suitesparse 4.5.4
+    - suitesparse 4.5.4   # [not win]
+    - suitesparse 5.1.2   # [win]
+    - vc 14   # [win and py35]
+    - vc 14   # [win and py36]
```

Comment:
These `vc` requirements also can be dropped for the same reason.

Reply:
There was a macOS-specific issue with suitesparse a while ago (conda-forge/suitesparse-feedstock#22). Aren't the suitesparse version pins necessary to avoid issues?

Reply:
Using 4.5.\* should be fine. It's been stable across minor versions. Same story with 5.1.\*. 5.1.0 added new functionality, but didn't break anything from 5.0.\*.

ref: https://abi-laboratory.pro/tracker/timeline/suitesparse/

Reply:
Moved [here]( https://github.com/conda-forge/cvxopt-feedstock/pull/17/files#r182202467 ).